third_party/terraform/tests/resource_monitoring_uptime_check_config_test.go

```diff
@@ -106,6 +106,7 @@ resource "google_monitoring_uptime_check_config" "http" {
 
   content_matchers {
     content = "example"
+	matcher = "CONTAINS_STRING"
```

Comment:
Spacing is off here, tabs vs spaces. After that this should be good to go

Reply:
done